doc/open-extensions/muc_light.md

```diff
@@ -962,6 +977,8 @@ The notifications contain a list of items. The item list may be different from t
 </message>
 ```
 
+Because `hag66` was not a user of this room before, they only receive **their own affiliation** and **no prev-version** element.
```

Comment:
Because `hag66` was not a member of this room before, they only receive **their own affiliation** and **no prev-version** element.